src/main/webapp/app/shared/notification/connection-notification/connection-notification.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from '@angular/core';
+import { AccountService } from 'app/core/auth/account.service';
+import { JhiWebsocketService } from 'app/core/websocket/websocket.service';
+import { User } from 'app/core/user/user.model';
+import { Alert, AlertService } from 'app/core/util/alert.service';
+
+@Injectable({ providedIn: 'root' })
+export class ConnectionNotificationService {
+    alert?: Alert;
+    connected?: boolean;
+
+    constructor(private accountService: AccountService, private jhiWebsocketService: JhiWebsocketService, private alertService: AlertService) {}
+
+    init() {
+        this.accountService.getAuthenticationState().subscribe((user: User | undefined) => {
+            if (user) {
+                // listen to connect / disconnect events
+                this.jhiWebsocketService.enableReconnect();
+                this.jhiWebsocketService.bind('connect', this.onConnect);
+                this.jhiWebsocketService.bind('disconnect', this.onDisconnect);
+            } else {
+                // On logout, reset component
+                this.connected = undefined;
+                this.alert?.close();
+                this.alert = undefined;
+                this.jhiWebsocketService.disableReconnect();
+                this.jhiWebsocketService.unbind('connect', this.onConnect);
+                this.jhiWebsocketService.unbind('disconnect', this.onDisconnect);
+            }
+        });
+    }
+
+    /**
+     * Only update on connect if there is not already an active connection.
+     * This alert is temporary and disappears after 10 seconds.
+     **/
+
```

Comment:
```suggestion
     */
```

Reply:
Hmm. I don't get these two. Why would we only want to put one star here?

Reply:
I think the usual formatting convention for both Javadoc and JSDoc is the double star at the start but only one at the end.

Reply:
Alright, I removed them